File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/filmDetailInfo/Rating.java
package com.rikkimikki.torrentino.domain.pojo.filmDetailInfo;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Rating {
    @SerializedName("kinopoisk")
    @Expose
    private KinopoiskReviews kinopoiskReviews;

    public KinopoiskReviews getKinopoiskReviews() {
        return kinopoiskReviews;
    }

    public void setKinopoiskReviews(KinopoiskReviews kinopoiskReviews) {
        this.kinopoiskReviews = kinopoiskReviews;
    }
}

File: app/src/main/java/com/rikkimikki/torrentino/domain/pojo/filmDetailInfo/Rating.kt
package com.rikkimikki.torrentino.domain.pojo.filmDetailInfo

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Rating (
    @SerializedName("kinopoisk")
    @Expose
    val kinopoiskReviews: KinopoiskReviews,
    val libriaReviews:Int=0
)